client-app/ui-kit/components/atoms/product-title/vc-product-title.vue

```diff
@@ -57,6 +60,20 @@ const linkTo = computed(() => (!props.disabled ? props.to : ""));
 
   @apply leading-[1.17] #{!important};
 
+  &--lines {
+    &--1 {
+      @apply line-clamp-1;
```

Comment:
Just wonder if we can use v-bind here, instead of setting `--lines` class?

https://vuejs.org/api/sfc-css-features#v-bind-in-css